README.md

```diff
@@ -14,6 +14,9 @@ To transform this skeleton in your plugin your may want to:
 * Create test input data in the folder [`testdata/input`](testdata/input)
   (refer to the [SDK manual for more details on how to define test data](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/concepts/test_framework.html))
 * Implement your plugin `process()` logic in [`plugin.py`](plugin.py)
+* implement your own @transformers in the example_transformer() [`plugin.py`](plugin.py)
```

Comment:
I'd suggest the following:
* Start sentence with a capital,
* Mention that step is, or that transformers are optional
* Remove the text about adding a tag, since the 'tag' is already in the template (n.b. in python it is usually referred as decorating a function or class, instead of tagging it.
* Make the first mention of transformer a link to the page in the extraction plugin SDK documentation that describes this transformer.

```suggestion
* (Optional) Implement your own [transformers](link to the external documentation) in the example_transformer() [`plugin.py`](plugin.py), or remove the transformer method from the plugin.
```